.scripts/interactive_translate/main.py

```diff
@@ -0,0 +1,224 @@
+        wrapped_text = textwrap.fill(response.text, width=72)
+        return wrapped_text
+
+    def compose(self) -> ComposeResult:
+        yield Header()
+        yield Container(
+            Static(id="status-line"),
+            id="status-container"
+        )
+        yield Container(
+            Static("Source Text", classes="section-label"),
+            Static(id="source-text", classes="text-content"),
+            Static("Raw Translation", classes="section-label"),
+            Static(id="raw-translation", classes="text-content"),
+            Static("New Translation", classes="section-label"),
+            Input(classes="translation-input", id="translation-input"),
+            Container(
+                Button("Previous [←]", id="prev-btn"),
+                Button("Accept [↵]", id="accept-btn"),
+                Button("Next [→]", id="next-btn"),
+                Button("Generate [^G]", id="generate-btn"),
+                Button("Save [^S]", id="save-btn", variant="primary"),
+                id="button-container"
+            ),
+            id="translation-container"
+        )
+        yield Footer()
+
+    def on_mount(self) -> None:
+        self.show_current_entry()
+
+    def show_current_entry(self) -> None:
+        if self.current_entry_index < len(self.po_file):
+            entry = self.po_file[self.current_entry_index]
+            status = f"File: {self.po_file_path.name} | Entry: {self.current_entry_index + 1}/{len(self.po_file)}"
+            self.query_one("#status-line").update(status)
+            self.query_one("#source-text").update(entry.msgid)
+            self.query_one("#raw-translation").update(entry.msgstr)
+            
+    async def on_button_pressed(self, event: Button.Pressed) -> None:
+        if event.button.id == "accept-btn":
+            translation = self.query_one("#translation-input").value
+            if translation:
+                self.po_file[self.current_entry_index].msgstr = translation
+                self.navigate_entry(1)
+        elif event.button.id == "next-btn":
+            self.navigate_entry(1)
+        elif event.button.id == "prev-btn":
+            self.navigate_entry(-1)
+        elif event.button.id == "generate-btn":
+            entry = self.po_file[self.current_entry_index]
+            try:
+                prompt = ("Translate the following Python documentation into Traditional Chinese"
+                f"for {self.po_file_path}:{entry.occurrences} with message {entry.msgid}. Ensure "
+                "that the translation is accurate and uses appropriate technical terminology. The"
+                " output must be in Traditional Chinese. Pay careful attention to context, idiomatic"
+                " expressions, and any specialized vocabulary related to Python programming. Maintain "
+                "the structure and format of the original documentation as much as possible to ensure"
+                " clarity and usability for readers.")
```

Comment:
```suggestion
                prompt = ("Translate the following Python documentation into Traditional Chinese "
                f"for {self.po_file_path}:{entry.occurrences} with message {entry.msgid}. Ensure "
                "that the translation is accurate and uses appropriate technical terminology. The "
                "output must be in Traditional Chinese. Pay careful attention to context, idiomatic "
                "expressions, and any specialized vocabulary related to Python programming. Maintain "
                "the structure and format of the original documentation as much as possible to ensure "
                "clarity and usability for readers.")
```
This has the same bug (`Chinesefor`) as the other prompt.  Since it looks duplicated, it might be better to factor it out, and reusing it in both places instead.